clang/lib/Driver/ToolChains/SYCL.cpp

```diff
@@ -552,10 +554,8 @@ void SYCL::fpga::BackendCompiler::ConstructJob(
     const char *FolderName = Args.MakeArgString(FN);
     ReportOptArg += FolderName;
   } else {
-    // Output directory is based off of the first object name as captured
-    // above.
-    if (!CreatedReportName.empty())
-      ReportOptArg += CreatedReportName;
+    // Default output directory should match default output executable name
+    ReportOptArg += "a.prj";
```

Comment:
`a.out` is the default for Linux.  This isn't the case for Windows is that OK?

Reply:
I did not realize that that would be the case. Thanks for pointing that out!

I would prefer to have it also be "a.prj" for Windows, so I'll look into that.